components/usage-api/usage/v1/usage.proto

```diff
@@ -10,6 +10,9 @@ service UsageService {
     // ListBilledUsage retrieves all usage for the specified attributionId
     rpc ListBilledUsage(ListBilledUsageRequest) returns (ListBilledUsageResponse) {}
 
+    // ListBilledUsage retrieves all usage for the specified attributionId
+    rpc FindUsage(FindUsageRequest) returns (FindUsageResponse) {}
```

Comment:
If we followed CRUD more closely then the RPC would generally be `Get/List/Create/Delete/Update` but I'm not too fussed here. It mostly stands out given the RPC above uses `List`